File: CS/GridControlCRUDMVVMInfiniteAsyncSource/IssuesContext.DataModel/IssueData.cs
using System;
using System.Linq;

namespace GridControlCRUDMVVMInfiniteAsyncSource {
    public class IssueData {
        public IssueData() {
            Created = DateTime.Today;
        }
        public int Id { get; set; }
        public string Subject { get; set; }
        public DateTime Created { get; set; }
        public int Votes { get; set; }
        public Priority Priority { get; set; }
        public int UserId { get; set; }
    }
}

File: VB/GridControlCRUDMVVMInfiniteAsyncSource/IssuesContext.DataModel/IssueData.vb
Imports System
Imports System.Linq

Namespace GridControlCRUDMVVMInfiniteAsyncSource
	Public Class IssueData
		Public Sub New()
			Created = DateTime.Today
		End Sub
		Public Property Id() As Integer
		Public Property Subject() As String
		Public Property Created() As DateTime
		Public Property Votes() As Integer
		Public Property Priority() As Priority
		Public Property UserId() As Integer
	End Class
End Namespace
